libs/libcommon/src/libcommon/processing_graph.py

```diff
@@ -17,6 +17,7 @@ class _ProcessingStepSpecification(TypedDict):
 class ProcessingStepSpecification(_ProcessingStepSpecification, total=False):
     requires: Optional[str]
     required_by_dataset_viewer: Literal[True]
+    version: str
```

Comment:
should we move to int? see below

Reply:
I renamed it to job_runner_version to be consistent with the new field